.github/files/generate-ci-matrix.php

```diff
@@ -65,29 +65,18 @@
 		'script'  => 'test-php',
 		'php'     => $php,
 		'wp'      => 'latest',
-		'timeout' => 20, // 2023-08-17: Successful runs seem to take up to ~12 minutes.
+		'timeout' => 20, // 2024-11-12: Successful runs seem to take up to ~7 minutes.
 	);
 }
 
-// TODO: When WordPress 6.5 is no longer supported, this can be removed. Runs too slow on ubuntu-24.04 (ubuntu-latest).
-$matrix[] = array(
-	'name'                => 'PHP tests: PHP 7.0 WP previous',
-	'runner'              => 'ubuntu-22.04',
-	'script'              => 'test-php',
-	'php'                 => '7.0',
-	'wp'                  => 'previous',
-	'timeout'             => 20, // 2023-08-17: Successful runs seem to take up to ~12 minutes.
-	'force-package-tests' => true,
-);
-
 foreach ( array( 'previous', 'trunk' ) as $wp ) {
 	$phpver   = $versions['PHP_VERSION'];
 	$matrix[] = array(
 		'name'    => "PHP tests: PHP {$phpver} WP $wp",
 		'script'  => 'test-php',
 		'php'     => $phpver,
 		'wp'      => $wp,
-		'timeout' => 15, // 2021-01-18: Successful runs seem to take ~8 minutes for the 7.4 trunk run, ~5.5-6 for 7.x and 8.0.
+		'timeout' => 15, // 2024-11-12: Successful runs seem to take ~7 minutes for the 7.4 trunk run, ~5.5-6 for 7.x and 8.0.
```

Comment:
If you're updating comments, this one should refer to the 8.2 WP trunk and 8.2 WP previous runs rather than what it's saying now.

Reply:
I actually meant to switch it to say "7 minutes" period, but apparently failed to chop the last part.

Reply:
And now we wait for CI again: 2ce5386339